components/runners/ambient-runner/ambient_runner/bridges/claude/grpc_transport.py

```
@@ -311,6 +311,34 @@ async def _handle_user_message(self, msg: Any) -> None:
+        try:
+            await _run_once()
+        except PermissionError as exc:
+            logger.warning(
+                "[GRPC LISTENER] Credential auth failure, refreshing token and retrying: session=%s error=%s",
+                self._session_id,
+                exc,
+            )
+            try:
+                from ambient_runner.platform.utils import refresh_bot_token
+                refresh_bot_token()
+            except Exception as refresh_exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)